source/sass/components/article-page.scss

```diff
@@ -332,6 +332,7 @@
   }
 }
 
+.article-image-block,
 .double-image-block,
 .full-width-image-block {
   figurecaption p {
```

Comment:
might wanna add `figcaption p` here as a selector first

I'll file a new ticket to replace all `figurecaption` instances with `figcaption` 😂